### PR TITLE
Move all code to a class, add unit tests, handle multiple pages of milestones and labels

### PR DESCRIPTION
**Pull request by williamdes**
- Moved all code to a class
- Added unit tests
- Fix to handle multiple pages of milestones and labels (#3)
- Added some properties into composer.json
- Upgraded dependencies
- Improved code by removing hard coded sql insert statements
- Added declaration for `github-to-mysql` into `composer.json` binary section, see: 8a0980ef4c2c8c6fd35a693e5bcbef24b3bfdb45

To review, you can review commit by commit

### CODE REVIEW

**Comment by williamdes** [comment]
Tested using `./github-to-mysql sync --since-forever phpmyadmin/phpmyadmin` since this command would not succeed before this PR, see: #3

**Comment by williamdes** [comment]
@mnapoli Can you please review my work ?

**Comment by mnapoli** [comment]
@williamdes I have been removed from this repository and I have no longer rights to maintain it. I don't know if this project is maintained, an option might be to use your own fork.

**Comment by williamdes** [comment]
cc @jeremy-wizaplace @luxifer

**Comment by luxifer** [comment]
Neither do I

**Comment by williamdes** [comment]
@anthHugo?

**Comment by steevanb** [comment]
We are waiting our CTO to know if we continue to maintain this repository :)

File: app/data.php [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace GitHubToMysql;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Exception\UniqueConstraintViolationException;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\ClientException;
+
+class data {
```
Comment (wizacedric) [inline app/data.php]: Let's follow a few principles here: 
- uppercase names for classes
- separation of concerns
- useful constructors

I suggest splitting this class in 2, one for database write operations, and one for http fetch operations.

File: composer.json [hunk 2]
```diff
@@ -1,8 +1,53 @@
 {
+    "name": "github-to-mysql",
```
Comment (wizacedric) [inline composer.json]:
```suggestion
    "name": "wizaplace/github-to-mysql",
```

File: composer.json [hunk 3]
```diff
@@ -1,8 +1,53 @@
...
+    "description": "Synchronizes GitHub data (issues, labels, ...) to a MySQL database.",
+    "homepage": "https://github.com/wizaplace/github-to-mysql",
+    "type": "library",
+    "readme": "https://github.com/wizaplace/github-to-mysql#readme",
+    "license": "UNLICENCED",
```
Comment (wizacedric) [inline composer.json]:
```suggestion
    "license": "MIT",
```

**Review by wizacedric — CHANGES_REQUESTED** [review]

**Comment by wizacedric** [comment]
@williamdes thanks for your work on your PR. These are some very good improvements, but I think you can go a bit further :rocket: .

File: composer.json [hunk 4]
```diff
@@ -1,8 +1,53 @@
 {
+    "name": "github-to-mysql",
+    "description": "Synchronizes GitHub data (issues, labels, ...) to a MySQL database.",
+    "homepage": "https://github.com/wizaplace/github-to-mysql",
+    "type": "library",
```
Comment (wizacedric) [inline composer.json]:
```suggestion
    "type": "project",
```

**Review by wizacedric — CHANGES_REQUESTED** [review]

**Comment by williamdes** [comment]
@wizacedric I improved this PR using some of your principles
> I suggest splitting this class in 2, one for database write operations, and one for http fetch operations.

I will maybe do some work on this

**Comment by williamdes** [comment]
@wizacedric Would you accept to merge this PR ?

**Comment by williamdes** [comment]
@wizagael can you please review my PR?

**Comment by williamdes** [comment]
cc @wizacedric ..
I invested a lot of my free time, please review or merge

**Review by wizacedric — APPROVED** [review]

**Pull request merged**